test/unit/org/apache/cassandra/transport/TransportTest.java

```diff
@@ -129,13 +93,8 @@ public void testAsyncTransport() throws Throwable
 
     private void doTestTransport() throws Throwable
     {
-        SimpleClient client = new SimpleClient(nativeAddr.getHostAddress(), nativePort);
-        QueryHandler queryHandler = (QueryHandler) cqlQueryHandlerField.get(null);
-        cqlQueryHandlerField.set(null, new TransportTest.TestQueryHandler());
-        try
+        try (SimpleClient client = new SimpleClient(nativeAddr.getHostAddress(), nativePort))
         {
-            requireNetwork();
```

Comment:
why is this no longer needed? And how did you know? :)

Reply:
It is just moved to another place - https://github.com/datastax/cassandra/pull/1497/commits/08c6710e9f830481426c339b8933e86319e21164#diff-ee7b0b3026637d0a5505f3beb591d6f7b46ae0fef2eac98f60d7dd1884e04689R62

Reply:
In most of our tests we add the `requireNetwork` while we setup config for testing, so it makes sense to me to be there.

Reply:
:doh: I missed that! Thanks